SpatialGDK/Source/SpatialGDKEditor/Public/SpatialGDKEditorSettings.h

```diff
@@ -344,6 +344,19 @@ class SPATIALGDKEDITOR_API USpatialGDKEditorSettings : public UObject
 	static bool IsRegionCodeValid(const ERegionCode::Type RegionCode);
 	static bool IsManualWorkerConnectionSet(const FString& LaunchConfigPath);
 
+public:
+	UPROPERTY(EditAnywhere, config, Category = "Mobile", meta = (DisplayName = "Connect to a local deployment"))
+		bool bMobileConnectToLocalDeployment;
```

Comment:
Nit - while this file seems to do both, I think prefer to have this indented to the same level.